standard/expressions.md

```diff
@@ -679,7 +697,7 @@ When a function member with a parameter array is invoked in its expanded form wi
 >
 > *end example*
 
-When arguments are omitted from a function member with corresponding optional parameters, the default arguments of the function member declaration are implicitly passed.
+When arguments are omitted from a function member with corresponding optional parameters, the default arguments of the function member declaration are implicitly passed. (This can involve the creation of an unnamed *variable_reference*, as described above.)
```

Comment:
```suggestion
When arguments are omitted from a function member with corresponding optional parameters, the default arguments of the function member declaration are implicitly passed. (This can involve the creation of a storage location, as described above.)
```

Reply:
Action from 2023-04-19: Accept this suggestion to be consistent with the previous one.